src/patternfly/components/DataList/examples/DataList.md

```diff
@@ -971,7 +971,7 @@ When a list item includes more than one block of content, it can be difficult fo
             This text will wrap to the next line because it has the default behavior of the data list cell.
           </span>
         {{/data-list-cell}}
-        {{#> data-list-cell data-list-cell--modifier="pf-m-truncate"}}
+        {{#> data-list-cell data-list-cell--IsTruncate}}
```

Comment:
Doesn't look like these are working? Probably because hbs params need a value of some sort, and later in this doc, the param case doesn't match.

```suggestion
        {{#> data-list-cell data-list-cell--IsTruncate=true}}
```